src/zenml/orchestrators/utils.py

```diff
@@ -50,6 +51,35 @@ def create_tfx_pipeline(
     )
 
 
+def is_cached_step(execution_info: data_types.ExecutionInfo) -> bool:
+    """Returns the caching status of a tfx step.
+
+    Args:
+        execution_info: The execution info of a tfx step.
+
+    Returns:
+        The caching status of a tfx step as a boolean value.
+    """
+    repository = Repository()
+    metadata_store = repository.get_stack(
+        repository.active_stack_name
+    ).metadata_store
+
+    step_name = str(
+        execution_info.exec_properties["zenml-pipeline_parameter_name"]
```

Comment:
Instead of `zenml-pipeline_parameter_name` I would use the constants defined in `BaseMetadataStore`

```
from zenml.metadata_stores.base_metadata_store import INTERNAL_EXECUTION_PARAMETER_PREFIX , PARAM_PIPELINE_PARAMETER_NAME

step_name_property = INTERNAL_EXECUTION_PARAMETER_PREFIX + PARAM_PIPELINE_PARAMETER_NAME,
```